Sources/OutlineView/OutlineViewDragAndDrop.swift

```diff
@@ -0,0 +1,139 @@
+import Foundation
+import AppKit
+
+/// A protocol for use with `OutlineView`, implemented by an object (most likely
+/// the data source) to handle receiving items that are dragged into the `OutlineView`.
+@available(macOS 10.15, *)
+public protocol DropReceiver {
+    associatedtype DataElement: Identifiable
+    
+    /// Given a dragged `NSPasteboardItem`, attempts to read the data in the item
+    /// into a usable object for the `OutlineView`'s data source.
+    ///
+    /// - Parameter item: The Pasteboard item that is being dragged into the OutlineView
+    ///
+    /// - Returns: nil if the Pasteboard item is not readable by this receiver, or
+    ///   a tuple with the decoded item and its associated PasteboardType, which must
+    ///   be included in `acceptedTypes`
+    func readPasteboard(item: NSPasteboardItem) -> DraggedItem<DataElement>?
+    
+    /// Called continuously as a dragged item is moved over the `OutlineView`, this is
+    /// used to define the behavior of the `OutlineView` and the drag cursor.
```

Comment:
By convention the doc comment should start with a high level description of the method/property. The first line is most likely to appear un-truncated in the code autocomplete suggestion window.

Here, "Called continuously ..." is really important information, but is not the high level description of the method. I would swap the order of the two phrases.

> Defines the behavior of the `OutlineView` and the drag cursor when an item is being dragged. Called continuously as a dragged item is moved over the `OutlineView`.

Reply:
Looks good to me. Fixed here https://github.com/Sameesunkaria/OutlineView/pull/10/commits/9a66eb8d6576eb16ca2d046bf17d013ac32249e3